backend/src/utils/shuffleArray.ts

```diff
@@ -0,0 +1,10 @@
+function shuffleArray<T>(array: T[]): T[] {
```

Comment:
[...Array(100).keys()].sort(() => Math.random() - 0.5).splice(0,10) Please investigate this approach